### ISSUE TITLE
Minishift unable to add impersonation 'sudoer' role when try to provision OpenShift =< v1.5.1

### ISSUE THREAD

**Issue by praveenkumar** [minishift/minishift#1242]
We have now a bug because we are using v3.6 oc version to provision openshift < 3.6 and when we try to provision v1.5.1 it failed to add sudoer role.

Code which we introduced for check openshift version and then according to it select which part of kubeconfig should be part of minishift_kubeconfig is broken. https://github.com/minishift/minishift/blob/master/pkg/minishift/clusterup/clusterup.go#L118 we might remove this whole addition code-base since now 3.6 is default.

```
The server is accessible via web console at:
    https://192.168.42.117:8443

You are logged in as:
    User:     developer
    Password: <any value>

To login as administrator:
    oc login -u system:admin

Error during post cluster up configuration: Unable to add sudoer role
```

**Comment by hferentschik** [comment]
Good catch. 

@agajdosi here we have something we need dearly some integration tests as well. They will be long running, but we should try to make sure that we can provision earlier (major) versions of OpenShift.

**Comment by agajdosi** [comment]
@hferentschik +1. I was thinking about this also, this "minishift + OSO of nearly whatever version" seems like a problematic zone. If you have some ideas on this topic already, how test should like, how many versions etc, please create an issue. If not, I will create the issue later today or tomorrow.

**Comment by hferentschik** [comment]
Merged via pull request #1245
